openlane/common/drc.py

```diff
@@ -20,7 +20,18 @@
 from dataclasses import dataclass, field, asdict
 from typing import List, Optional, Tuple, Dict
 
-BoundingBox = Tuple[Decimal, Decimal, Decimal, Decimal]  # microns
+
+@dataclass
+class BoundingBox:
```

Comment:
Why? I wanted to add this to 2.3 but I can't because it's an API break now for some reason

Reply:
I noticed that `BoundingBox` is not part of the public API. But since it is a member of `Violation`, that would be an API break. I will restore the tuple then.